HW7/AntonKoluh/07-03-Outside/7-Mult3or5.py

```diff
@@ -0,0 +1,6 @@
+def solution(number):
+    num_lst = []
+    for x in range(number - 1, 0, -1):
```

Comment:
There's no need to use reverse order.
The code can be written as:
return sum(x for x in range(1, number) if x % 3 == 0 or x % 5 == 0)